src/services/users.service.ts

```diff
@@ -106,6 +106,15 @@ async function assignRole(id: string | number, role: Role): Promise<User> {
 }
 
 async function remove(ids: (string | number)[]): Promise<boolean> {
+
+  const userString = localStorage.getItem("user");
```

Comment:
service is responsible only for sending requests
could you move this into component as validation step?